pkg/ebpf/events_pipeline.go

```diff
@@ -427,7 +427,7 @@ func (t *Tracee) sinkEvents(ctx context.Context, in <-chan *trace.Event) <-chan
 		for event := range in {
 			// Only emit events requested by the user
 			id := events.ID(event.EventID)
-			if !t.events[id].emit {
+			if (t.events[id].emit & t.events[id].submit) == 0 {
```

Comment:
The AND should be between emit bitmap and matched_scopes (of the event) bitmap

Reply:
Changing that to `if (t.events[id].emit & event.MatchedScopes) == 0 {` doesn't output the sig event (scope 3), since it comes with scopes zeroed and don't output the ptrace (scope 1) as well.

![image](https://user-images.githubusercontent.com/3372117/214297790-967f9763-5e67-4090-bd2f-e45b69e5da80.png)

Reply:
There's the `FindingToEvent` function in `ebpf/finding.go`, you need to add that field to the converted event.